sdk/lib/drivers/sound/mmixer/controls.c

```diff
@@ -218,10 +218,10 @@ MMixerAddMixerControl(
                 }
 
                 Value = Range->Bounds.SignedMinimum;
-                for(Index = 0; Index < Steps; Index++)
+                for (Index = 0; Index < Steps; Index++)
                 {
                     VolumeData->Values[Index] = Value;
-                    Value += Range->SteppingDelta;
+                    Value += Range->SteppingDelta - 1;
```

Comment:
Suggestion to add such a comment, because without it the code reader wouldn't really understand why the "-1", instead of nothing, or some other value...
But I would also suggest you to retest what really happens without this "- 1" before we can merge this PR.
```suggestion
                    Value += Range->SteppingDelta - 1; // '- 1' to make the left and right volume controls behave independently.
```

Reply:
This -1 "hacks" a weird situation... That is not only affecting this side.
The situation is weeeeird as hell, because the calc of % are not ok (unable to reach 0 or 100) and sincronization between the task bar gain slicer, the sndvol32 and the properties audio mixer, and the sndvol32 horizontal and vertical bars having some "charity" between them. I am sure that there are lot of bugs behind every layer of software that moves this, so... I think I can help somehow debugging this.

For me this PR is ready to merge, as a first step to the audio GUI to not be embarrasing in terms of functionality. ReactOS needs a decent audio, let's go audio!

Reply:
OK, I also found out that this `-1` is actually a hackish solution. So I'll revert it once the proper fix is made by me. (or simply not commit it here? :thinking: )

Reply:
Added a proposed comment as well (but with a `HACK` prefix, since it's actually a hackish solution).